cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout-full.html" %}
+
+{% block hero %}
+    {% for block in page.header %}
+        {% if block.block_type == 'features' %}
```

Comment:
There's a use-case for `features` being added to this template?

<img width="1362" alt="Screen Shot 2021-05-27 at 1 49 58 PM" src="https://user-images.githubusercontent.com/704760/119873364-76d58580-bef2-11eb-9eda-bfbf12d4f473.png">

Reply:
The jumbo hero is designed to have optional features tucked underneath it. See [SSSU](https://www.consumerfinance.gov/start-small-save-up/) or the homepage. We won't be using it for our new inequity page but future story pages might need it.